### PR TITLE
Faster danger

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

**Comment by asfalcone** [comment]
This is "faster" because the results from danger like (Big PR, No PR description) are delivered to the PR faster because danger is run before the tests (and again after the tests). :+1:

File: Dangerfile [hunk 1]
```diff
@@ -12,24 +12,24 @@ require 'json'
 
 containing_dir = ENV["CIRCLE_ARTIFACTS"] || "." # for local testing
 rspec_files = Dir[File.join(containing_dir, "rspec_logs_*.json")]
-fail("Could not find test artifacts") if rspec_files.count == 0
+if rspec_files.count > 0
```
Comment (mfurtak) [inline Dangerfile]: The contents of the `each` iteration should be skipped if the count is zero either way. No check needed here! :+1:

Comment (KrauseFx) [inline Dangerfile]: Very good point :+1:

**Comment by KrauseFx** [comment]
Okay, thanks, simplified the PR, let's see how that works :+1:

**Comment by KrauseFx** [comment]
Okay, `danger` now only takes one minute to reply. I'm also working on enabling the RubyGems cache now!

**Comment by mfurtak** [comment]
:+1: :runner: :dash:

**Pull request merged**